.ci-orchestrator/pb.yml

```diff
@@ -347,7 +288,7 @@ steps:
 
 - stepName: Unittest Open Liberty
   workType: RTC
-  projectName: "Liberty Personal Build CI Orchestrator - EBC"
+  projectName: "Open Liberty Personal Build CI Orchestrator - EBC"
```

Comment:
Is this going to cause retention issues?

Reply:
Pruning policy of `Open Liberty Personal Build CI Orchestrator - EBC` is
```
Successful build results to keep: 20
Failed build results to keep: 10
```

versus pruning policy of `Liberty Personal Build CI Orchestrator - EBC` is
```
Successful build results to keep: 30
Failed build results to keep: 15
```

10 fewer successful builds and 5 fewer failed builds. Do you think we should increase that to match if there's disk availability?

Reply:
Lets deliver this but monitor over the next few days to see if we are deleting builds to eagerly